Integrations/ThreatX/ThreatX.py

```diff
@@ -0,0 +1,601 @@
+        'ip': ip
+    }
+
+    return http_request('/lists', commands)
+
+
+@logger
+def unwhitelist_ip_command():
+    ip = demisto.args().get('ip', None)
+    results = unwhitelist_ip(ip)
+
+    md = tableToMarkdown('Unwhitelist IP',
+                         results,
+                         ['Result'],
+                         removeNull=True)
+
+    ec = {
+        'IP(val.Address === obj.Address)': {
+            'Address': ip
+        }
+    }
+
+    return_outputs(md, ec, results)
+
+
+@logger
+def get_entities(entity_name, entity_id, entity_ip, timeframe):
+    commands = {
+        'command': 'list',
+        'query': dict()
+    }  # type: dict
+
+    if entity_name is not None:
+        entity_names = entity_name.split(',')
+        my_entity_name = {'codenames': entity_names}
+        commands['query'].update(my_entity_name)
+
+    if entity_id is not None:
+        entity_ids = entity_id.split(',')
+        my_entity_id = {'entity_ids': entity_ids}
+        commands['query'].update(my_entity_id)
+
+    if entity_ip is not None:
+        entity_ips = entity_ip.split(',')
+        my_entity_ip = {'ip_addresses': entity_ips}
+        commands['query'].update(my_entity_ip)
+
+    first_seen = None
+
+    if timeframe is not None:
+        if timeframe == '1-Hour':
+            first_seen = int(time.time() - (60 * 60))
+        elif timeframe == '1-Day':
+            first_seen = int(time.time() - (24 * 60 * 60))
+        elif timeframe == '1-Week':
+            first_seen = int(time.time() - (7 * 24 * 60 * 60))
+        elif timeframe == '1-Month':
+            first_seen = int(time.time() - (31 * 24 * 60 * 60))
+
+    if first_seen:
+        my_timeframe = {'first_seen': first_seen}
+        commands['query'].update(my_timeframe)
+
+    return http_request('/entities', commands)
+
+
+@logger
+def get_entity_risk(entity_id):
+    commands = {
+        'command': 'risk_changes',
+        'id': entity_id
+    }
+
+    return http_request('/entities', commands)
+
+
+@logger
+def get_entities_command():
+    entity_name = demisto.args().get('entity_name', None)
+    entity_id = demisto.args().get('entity_id', None)
+    entity_ip = demisto.args().get('entity_ip', None)
+    timeframe = demisto.args().get('timeframe', None)
+    results = get_entities(entity_name, entity_id, entity_ip, timeframe)
+    e_risk = None
+    e_id = None
+    md = []
+    ec = {}
+    dbot_scores = []
+    ip_enrich = []
+
+    for entity in results:
+        risk_score = 0
+
+        # Grab the entity risk so we can set the Dbot score for the Actor IPs
+        e_id = entity.get('id')
+
+        if e_id:
+            e_risk = get_entity_risk(e_id)
+
+        if isinstance(e_risk, list) and e_risk:
+            if isinstance(e_risk[-1], dict) and 'risk' in e_risk[-1]:
+                risk_score = e_risk[-1]['risk']
+
+        entity['risk'] = risk_score
+
+        iplist = []
+
+        if 'actors' in entity:
+            for actor in entity['actors']:
```

Comment:
remove the condition
```suggestion
            for actor in entity.get('actors', []):
```